### DISCUSSION TITLE
关于SMB协议 About SMB

### DISCUSSION THREAD

**Discussion by XIAOKU2300** [IceWhaleTech/CasaOS#1700] [Q&A]
我的旧电视机顶盒，不支持SMB 2.0 我需要将他的协议设置为SMB1.0 但是在CasaOS中没有找到SMB 协议调整的设置……请求各位帮助，谢谢

_My old TV set-top box doesn't support SMB 2.0. I need to set his protocol to SMB1.0, but I can't find the setting of SMB protocol adjustment in CasaOS ... Please help me, thank you._

**Comment by ChishFoxcat** [comment]
我在 `/etc/samba` 目录中发现有正常的 `smb.conf` 配置文件，但其中提示不允许修改
![QQ_1728341455774](https://github.com/user-attachments/assets/4ca0a562-e090-4457-bf46-82f563b2a260)

配置文件大概如下：
```yml
[global]
## fruit settings
   min protocol = SMB2 # 这个应该就是改最小协议的一行
   ea support = yes
```

可将 `min protocol = SMB2` 注释掉，增加 `server min protocol = NT1`
> 网络查询得此方法，未经验证

**Reply by XIAOKU2300** [reply]
可以，但是已经换其他操作系统了|･ω･｀)
